blog/README.md

```diff
@@ -0,0 +1,298 @@
+# CoreOS Layering
+
+In this blog we are going to explore the coreos-layering technology in order to
```

Comment:
Move the next 7 lines under Prerequisites

Reply:
Done.